frontend/appflowy_flutter/lib/plugins/database_view/widgets/card/container/card_container.dart

```diff
@@ -42,15 +40,10 @@ class RowCardContainer extends StatelessWidget {
             }
           }
 
-          return GestureDetector(
```

Comment:
Hold on, are we sure we can remove this `GestureDetector` and the `openCard` callback? This card widget is also used in Kanban Board in order to open the cards there.

Reply:
@richardshiue 
Oh my mistake,  While checking the `RowCardContainer`, I forgot to check the `RowCard` which is also utilized on the board page. I noticed Lucas rebased it and put it back to `RowCardContainer`. He deleted the tap function on `FlowyHover` and added `openCard` in its child widget `RowCard`. It solved the issue. 

Thanks for the help @LucasXu0